modules/gapi/src/backends/cpu/gcpuimgproc.cpp

```diff
@@ -335,6 +345,29 @@ GAPI_OCV_KERNEL(GCPUNV12toRGBp, cv::gapi::imgproc::GNV12toRGBp)
     }
 };
 
+G_TYPED_KERNEL(GConcatYUVPlanes, <cv::GMat(cv::GMat, cv::GMat)>, "concatyuvplanes") {
```

Comment:
I don't think the user needs this operation

Reply:
User doesn't need  it